frontend/src/scenes/trends/persons-modal/personsModalLogic.ts

```diff
@@ -299,16 +317,23 @@ export const personsModalLogic = kea<personsModalLogicType>([
                 return cleanFilters(filter)
             },
         ],
+        SelectFields: [
```

Comment:
Hmm, any specific reason to have this uppercase? We usually have most selectors/reducers/values lowercased. I'm also not sure why `ActorsQuery` below is uppercased 🤷

Reply:
> I'm also not sure why ActorsQuery below is uppercased 🤷

Aaand that is the one I cased it after... will adjust 👍🏻